### ISSUE TITLE
Works on pod v 1.8.3, doesn't work on pod v 1.9.1

### ISSUE THREAD

**Issue by ghost** [joncardasis/cocoapods-user-defined-build-types#3]
Hitting this error while running pod install in v1.9.1
[!] Invalid `Podfile` file: uninitialized constant Pod::Target::BuildType
Did you mean? Pod::BuildType.
 >  pod ‘FSCalendar’, :build_type => :dynamic_framework

I need to downgrade it to 1.8.3 and it works again. 
Putting the instructions here on how to downgrade in case anybody needs it
Check cocoapod version:
```
gem which cocoapods
sudo gem uninstall cocoapods
sudo gem install cocoapods -v 1.8.3 -n /usr/local/bin
pod deintegrate
pod install
```

**Comment by dudihisine** [comment]
You can also follow my instruction [here](https://github.com/joncardasis/cocoapods-user-defined-build-types/issues/1#issuecomment-620026565) to fix the problem and use version 1.9.1

**Comment by joncardasis** [comment]
Fixed in v0.0.7 via https://github.com/joncardasis/cocoapods-user-defined-build-types/commit/c9077a02f2b79e208b84a1da13a048131f5c92d6. The resolution will work for 1.8.3 and 1.9.1
